pdc_client/__init__.py

```diff
@@ -91,27 +91,71 @@ def read_config_file(server_alias):
 
 
 class PDCClient(object):
-    """BeanBag wrapper specialized for PDC access.
```

Comment:
I sometimes do this too, but try to change only relevant lines for the change. It's easier to review and "git blame" gives some sane results.

Reply:
Make sense to me, will change this back, just think that write as this line is very ugly.
Don't why write like this although I saw beanbag also write the first line of docstrings like this.